Charts.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
is this Package.resolved necessary to be commited? so everyone is with the same version of the packages?

Reply:
It is, for the reason you stated. Just like we commit a Gemfile.lock or Cartfile.resolved.